setup.py

```diff
@@ -92,13 +95,16 @@ def finalize_options(self):
       ext_modules = [module1, module2, module3],
       classifiers=[
             'License :: OSI Approved :: BSD License',
-            'Programming Language :: Python',
-            'Programming Language :: Python :: 2',
-            'Programming Language :: Python :: 3',
+            'Programming Language :: Python :: 3 :: Only',
```

Comment:
I know Python 2 is deprecated, but unsure if anyone is using it still. I hope it is safe to remove python 2 from build here for this version.

Reply:
I think there are numerous dependencies here which do not support Python 2. Today, most libraries only support Python >= 3.6.

Python 2 is also no longer developed or maintained, so I would certaintly remove support for it in 2022. However, there might be some that still use older versions of Python 3. Up to you and the SimpleCRF team if this is something you want.